Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -516,8 +516,9 @@ GDCMImageIO::InternalReadImageInformation()
         // Errorneus file CT-MONO2-8-abdo.dcm,
         // the spacing is something like that [0.2\0\0.200000]
         // TODO: throw an expection that VM is not compatible
-        m_El.SetLength(4);
+        m_El.SetLength(16);
```

Comment:
The value `m_El.SetLength( entry.GetVM().GetLength() * entry.GetVR().GetSizeof() );`
is from dictionary and always 2*8